client/stylesheets/_toCat.scss

```diff
@@ -10,9 +10,9 @@
 GLOBAL | GLOBAL DIVS | HEADER | FOOTER **/
 
 /****** GLOBAL ******/
-
+ 
 ::selection {
-    background-color: yellow;
+    background-color: rgba(255, 255, 0, 0.67);
```

Comment:
Color literals like `rgba(255, 255, 0, 0.67)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces